LICENSE-Apache-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
What's this about?

Reply:
The GCP protobuf definitions are copied straight out of Google's public repository, and they carry the APL on them, which requires the presence of that license file.

Reply:
I'd expect we'd need to make it clear it just applies to the protobuf files, no? (of course I know nothing about license law)

Reply:
Yes, we should place this in the notices file or somewhere that is not the root of the project. Otherwise it'll get confused with out own license.

Reply:
Would it be worthwhile to try to rewrite the proto definitions from the online docs instead of using this (minimized) copy from the protobuf repository? Another option might be to pull in the API repository as a git submodule, although this would be the precedent for that.